src/java.desktop/share/classes/java/awt/color/ICC_Profile.java

```diff
@@ -1128,11 +1059,9 @@ public int getProfileClass() {
     byte[] theHeader;
     int theClassSig, theClass;
 
-        if (deferralInfo != null) {
-            return deferralInfo.profileClass; /* Need to have this info for
-                                                 ICC_ColorSpace without
-                                                 causing a deferred profile
-                                                 to be loaded */
+        ProfileDeferralInfo info = deferralInfo;
+        if (info != null) {
+            return info.profileClass;
```

Comment:
Separate MT issue found by the code review, we can get NPE here if deferralInfo will be set to null(activated) after the null-check. Proved by the ProfileActivationDuringpropertyAccess test.